Running "main" (2 tests)...
words_test.c:13: Failure: words_tests -> returns_word_count 
	Expected [word_count] to [equal] [4]
		actual value:			[2]
		expected value:			[4]

Completed "words_tests": 1 pass, 1 failure, 0 exceptions.
Completed "main": 1 pass, 1 failure, 0 exceptions.
